website/docs/r/repository_collaborators.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "github"
+page_title: "GitHub: github_repository_collaborators"
+description: |-
+  Provides a GitHub repository collaborators resource.
+---
+
+# github_repository_collaborators
+
+Provides a GitHub repository collaborators resource.
+
+This resource allows you to manage all collaborators for repositories in your
+organization or personal account. For organization repositories, collaborators can
+have explicit (and differing levels of) read, write, or administrator access to 
+specific repositories, without giving the user full organization membership. 
+For personal repositories, collaborators can only be granted write
+(implicitly includes read) permission. 
+
+When applied, an invitation will be sent to the user to become a collaborators
+on a repository. When destroyed, either the invitation will be cancelled or the
+collaborators will be removed from the repository.
+
+Further documentation on GitHub collaborators:
+
+- [Adding outside collaborators to your personal repositories](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories)
+- [Adding outside collaborators to repositories in your organization](https://help.github.com/articles/adding-outside-collaborators-to-repositories-in-your-organization/)
+- [Converting an organization member to an outside collaborators](https://help.github.com/articles/converting-an-organization-member-to-an-outside-collaborator/)
+ 
+~> Note: github_repository_collaborators cannot be used in conjunction with github_repository_collaborator and 
```

Comment:
Can this note be bolded and put above the docs links? ?

Reply:
Sure thing, I also added the same note to the non-authoritative resource, and added a line as well to clarify further.